docs/api-reference/Admin.md

```diff
@@ -1,29 +1,102 @@
 
 # quilt3.admin
+
 APIs for Quilt administrators. 'Registry' refers to Quilt stack backend services, including identity management.
 
-## create\_user(\*, username: str, email: str)  {#create\_user}
+
+## ManagedRole(\*, \_\_typename: Literal['ManagedRole'], id: str, name: str, arn: str) -> None  {#ManagedRole}
+Create a new model by parsing and validating input data from keyword arguments.
+
+Raises [`ValidationError`][pydantic_core.ValidationError] if the input data cannot be
+validated to form a valid model.
+
+`self` is explicitly positional-only to allow `self` as a field name.
+
+## UnmanagedRole(\*, \_\_typename: Literal['UnmanagedRole'], id: str, name: str, arn: str) -> None  {#UnmanagedRole}
+Create a new model by parsing and validating input data from keyword arguments.
+
+Raises [`ValidationError`][pydantic_core.ValidationError] if the input data cannot be
+validated to form a valid model.
+
+`self` is explicitly positional-only to allow `self` as a field name.
+
+## User(\*, name: str, email: str, date\_joined: datetime.datetime, last\_login: datetime.datetime, is\_active: bool, is\_admin: bool, is\_sso\_only: bool, is\_service: bool, role: Optional[Annotated[Union[quilt3.admin.ManagedRole, quilt3.admin.UnmanagedRole], FieldInfo(annotation=NoneType, required=True, discriminator='typename\_\_')]], extra\_roles: List[Annotated[Union[quilt3.admin.ManagedRole, quilt3.admin.UnmanagedRole], FieldInfo(annotation=NoneType, required=True, discriminator='typename\_\_')]]) -> None  {#User}
+Create a new model by parsing and validating input data from keyword arguments.
+
+Raises [`ValidationError`][pydantic_core.ValidationError] if the input data cannot be
+validated to form a valid model.
+
+`self` is explicitly positional-only to allow `self` as a field name.
```

Comment:
That's not great but might be OK for now.
I don't think we should try to do something with https://github.com/quiltdata/pydoc-markdown/ and should switch to something something commonly used e.g. sphinx which has extension for pydantic: https://sphinx-pydantic.readthedocs.io/en/latest/

Reply:
> switch to something something commonly used e.g. sphinx which has extension for pydantic

yeah that would be nice